contents/docs/llm-analytics/installation/openai.mdx

```diff
@@ -12,6 +44,12 @@ npm install openai
 
 </MultiLanguage>
 
+<LLMsSDKsCallout />
```

Comment:
Should this be under the `Install the PostHog SDK` section? Or maybe this should be used where we show how to use these SDKs?

This is pretty out of context for someone to understand what `These SDKs` refer to.

Reply:
+1